analytics_data_api/v0/models.py

```diff
@@ -493,6 +493,12 @@ def get_timeline(self, course_id, username):
 
         return full_timeline
 
+    def get_simple_data_for_all_students(self, course_id):
+        queryset = ModuleEngagement.objects.all().filter(course_id=course_id) \
+            .order_by('username')
+
+        return [entry for entry in queryset]
```

Comment:
I think you can/should just return the queryset here instead of resolving it directly to a python list.

related: I would recommend putting pagination of some kind on this API. IIRC that's easier to do if this is a queryset instead of a list.

Reply:
I don't think pagination makes sense in our use case as we our exporting all data into .csv, not displaying it on screen.

Reply:
There's no harm in returning the queryset here. In fact it would be clearer.